Makefile.ci.toml

```diff
@@ -253,13 +253,29 @@ category = "CI - INTEGRATION TESTS"
 command = "cargo"
 args = [
     "llvm-cov",
-    "nextest",
-    "--html",
+    "--lcov",
+    "--output-path",
+    "lcov.info",
     "--locked",
     "--no-default-features",
     "--features",
     "allocator,allocation-tracking,storage-mem,scripting,http,jwks",
     "--workspace",
+    "--",
```

Comment:
This was inadvertently removed in a previous PR that updated code coverage:
https://github.com/surrealdb/surrealdb/commit/9a555321a16cafee60960befa3fb8698394f3b97#diff-d3d6602e71abbfeae429c840c9f548cde3fb58bc067fc3358b3ad6828bf37815